src/main/java/pages/WelcomePage.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+import utils.MobileManagement;
+
+public class WelcomePage extends BasePage {
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.view.ViewGroup[@content-desc=\"continueAsGuest\"]/android.widget.TextView")
```

Comment:
could also have used uiautomator: new UiSelector().text("Continue as Guest"), but if we change to espresso(surprise) we will have to change it. its ok for now